static/src/js/middleware/metrics/__tests__/helpers.test.js

```diff
@@ -34,6 +34,7 @@ describe('helpers', () => {
     })
   })
 
+  // TODO these data structures don't seem right they're arrays in the console
```

Comment:
For the other tests these were arrays should just update them to better match what we are seeing during run-time